### PR TITLE
docs: document k8s NP missing features

### PR DESCRIPTION
**Pull request by aanm**
Signed-off-by: André Martins <andre@cilium.io>

### CODE REVIEW

**Comment by aanm** [comment]
test-me-please

File: Documentation/kubernetes/policy.rst [hunk 1]
```diff
@@ -28,6 +28,18 @@ NetworkPolicy
 For more information, see the official `NetworkPolicy documentation
 <https://kubernetes.io/docs/concepts/services-networking/network-policies/>`_.
 
+Known missing features for Kubernetes Network Policy:
+
++---------------------------------------------------------------------------+
+| Feature                    | Tracking Issue                               |
++===========================================================================+
```
Comment (ianvernon) [inline Documentation/kubernetes/policy.rst]: Can you test the rendered output? I think you need plus signs above and below `|` next to `Tracking Issue` and to the left of the GitHub links.

**Comment by aanm** [comment]
test-me-please

**Comment by tgraf** [comment]
test-me-please

**Comment by eloycoto** [comment]
test-me-please

**Review by ianvernon — APPROVED** [review]

**Pull request merged**